### DISCUSSION TITLE
Unable to see wyze cameras

### DISCUSSION THREAD

**Discussion by rAf19** [mrlt8/docker-wyze-bridge#901] [Q&A]
Good morning.  I updated my wyze bridge to 2.3.8 and I added the API Key/ID to my stack in Portainer but I keep receiving the messages below.  I also have two-factor authentication enabled (through TOTP) and have added 'MFA_TYPE=TotpVerificationCode' to the stack but still not working.

🚀 DOCKER-WYZE-BRIDGE v2.3.8 
[WyzeBridge] 🔍 Could not find local cache for 'auth'
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
 * Serving Flask app 'frontend'
 * Debug mode: off
[WyzeBridge] WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://172.19.0.2:5000
[WyzeBridge] Press CTRL+C to quit
[WyzeBridge] ⚠️ ERROR: HTTPSConnectionPool(host='auth-prod.api.wyze.com', port=443): Max retries exceeded with url: /api/user/login (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f6166ec4d50>: Failed to resolve 'auth-prod.api.wyze.com' ([Errno -3] Temporary failure in name resolution)"))
[WyzeBridge] 🔍 Could not find local cache for 'user'
[WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
[WyzeBridge] 🔍 Could not find local cache for 'auth'
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...

Below is the stack I am using:
version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554 # RTSP
            - 8888:8888 # HLS
            - 8889:8889 #WebRTC
            - 8189:8189/udp # WebRTC/ICE
            - 5080:5000 # WEB-UI
        environment:
            # [REQUIRED] Fill the following two lines:
            - WYZE_EMAIL=*myEMAIL*
            - WYZE_PASSWORD=*mypassword*
            - API_ID=*myID*
            - API_KEY=*myKEY*
	    - MFA_TYPE=TotpVerificationCode

**Comment by rAf19** [comment]
So when I create the container using yaml file (either through Container Manager in Synology or through a stack in Portainer), I kept receiving this '[WyzeBridge] ⚠️ ERROR: HTTPSConnectionPool(host='auth-prod.api.wyze.com', port=443): ' message.

However, when I go into Container Manager, download the mrlt8 image file, and manually set it up from the beginning, it works fine and I am able to see the cameras just fine.

Seems like I am missing a step or something when using a .yaml file but glad it works now.

**Reply by mrlt8** [reply]
Interesting, will have to look into that further!
